src/ThirdParty/ThirdParty.router.spec.ts

```diff
@@ -93,4 +95,38 @@ describe('ThirdParty router', () => {
         })
     })
   })
+
+  describe('when retreiving the available slots for a third party', () => {
+    let url: string
+    const maxSlots = 10
+    const itemsInCuration = 6
+
+    beforeEach(() => {
+      ;(thirdPartyAPI.fetchMaxItemsByThirdParty as jest.Mock).mockResolvedValueOnce(
+        maxSlots
+      )
+      ;(thirdPartyAPI.isManager as jest.Mock).mockResolvedValueOnce(true)
+      ;(ItemCuration.getItemCurationCountByThirdPartyId as jest.Mock).mockResolvedValueOnce(
+        [{ count: itemsInCuration }]
+      )
+      url = '/thirdParties/aThirdPartyId/slots'
+    })
+
+    afterEach(() => {
+      jest.restoreAllMocks()
```

Comment:
If this is being run in a top-level describe of the same scope, it shouldn't be necessary here!